src/create-embed.js

```diff
@@ -67,7 +67,7 @@ const createIframeEmbed = (params) => {
  *         The DOM element that will ultimately be passed to the `bc()` function.
  */
 const createInPageEmbed = (params) => {
-  const {embedOptions} = params;
+  const {embedOptions, options} = params;
```

Comment:
I don't think we would want to add a new `options` property when we already have a `embedOptions` property that serves the same purpose.

Reply:
I don't think this should be moved to embedOptions. `controls: Boolean` [is video.js option](https://docs.videojs.com/tutorial-options.html#controls), original issue appears because player-loader create controls attribute without respecting passed options, and attribute takes precedence over options object. It's [already noted in README](https://github.com/brightcove/player-loader#options) that options object will be passed to video.js.